doc/man/8/radosgw-admin.rst

```diff
@@ -434,17 +434,17 @@ Options
 
 .. option:: --min-rewrite-size
 
-    Specify the min object size condition for bucket rewrite (default 4M).
+    min object size for bucket rewrite (default 4M).
```

Comment:
looks like all the other option descriptions are capitalized. and a lot of them start with 'Specify', so maybe keep that part too? up to you

Reply:
@cbodley Done. Please verify. Thanks.